### ISSUE TITLE
Outside Editor, Unium web server binds to a single interface only

### ISSUE THREAD

**Issue by partlyhuman** [gwaredd/unium#15]
This appears to be by design but the behaviour is not desired in my application. In the editor the Server uses IPAddress.Any, which lets me connect to the Unium instance on

- localhost
- 127.0.0.1
- any loopback address (127.0.0.*)
- hostname
- its public IP

However due to line 117 of `UniumComponent.cs`, in a build, I can only connect to the unium instance at the IP detected by `Util::GetIPAddress()`. Besides being hard to always look up the external IP, this creates problems when there are multiple network interfaces (for instance ethernet + wifi) and we need to use ethernet on the local subnet but it grabs the IP from the wifi connection instead.

I can fix this by commenting this out and leaving IPAddress.Any, but I think it might be better to leave it that way by default, or expose the choice as an option.

**Comment by gwaredd** [comment]
Yes I think leaving it as any by default is probably the best option. If you create a pull request I can merge it in, otherwise I'll take a look next week when I'm back in the office. 

Cheers

**Comment by gwaredd** [comment]
Thanks :)
